views/editor/projects.html

```diff
@@ -29,10 +29,11 @@
           <td class="project-title">
             <a target="_self" href="/{{locale}}/user/{{username}}/{{project.id}}/{{queryString}}"> 
               {{project.title}}
-              <span class="project-information"></span>
+              <span class="project-information"></span><br />
```

Comment:
Use a `<div>` if you need it to be a block element, and get rid of the `<br />`.